Exploding on false webhooks

If a task is submitted with a webhook that routes to a non-existing server it crashes.

```
➜  jobukyu git:(master) npm start

> jobukyu@0.0.2 start [REDACTED]/jobukyu
> node index.js

Jobukyu is listening on port 3800
POST /jobs 201 30ms
GET /jobs/search?status=new&type=jobukyu_test_getjobfromqueue 200 11ms
PUT /jobs/52f4f2c0272fd78c4c000001/take 201 10ms
POST /jobs 201 12ms
POST /jobs 201 10ms
POST /jobs 201 8ms
GET /jobs/search?status=new&type=jobukyu_test_waitforjob_take 200 4ms
PUT /jobs/52f4f2c0272fd78c4c000002/take 201 7ms
GET /jobs/search?status=new&type=jobukyu_test_waitforjob_complete 200 3ms
PUT /jobs/52f4f2c0272fd78c4c000003/take 201 10ms
PUT /jobs/52f4f2c0272fd78c4c000003/complete 201 11ms
GET /jobs/search?status=new&type=jobukyu_test_waitforjob_error 200 3ms
PUT /jobs/52f4f2c0272fd78c4c000004/take 201 6ms

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: getaddrinfo ENOTFOUND
    at errnoException (dns.js:37:11)
    at Object.onanswer [as oncomplete] (dns.js:124:16)

npm ERR! jobukyu@0.0.2 start: `node index.js`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the jobukyu@0.0.2 start script.
npm ERR! This is most likely a problem with the jobukyu package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node index.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls jobukyu
npm ERR! There is likely additional logging output above.
npm ERR! System Linux 3.8.0-35-generic
npm ERR! command "[REDACTED]/v0.10.24/bin/node" "[REDACTED]/v0.10.24/bin/npm" "start"
npm ERR! cwd [REDACTED]/jobukyu
npm ERR! node -v v0.10.24
npm ERR! npm -v 1.3.21
npm ERR! code ELIFECYCLE
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     [REDACTED]/jobukyu/npm-debug.log
npm ERR! not ok code 0

```

--- a/lib/models/job.js
+++ b/lib/models/job.js
@@ -28,28 +28,28 @@ function processWebhook (webhook, metadata) {
   {
   case 'post':
     request.post({url: webhook.url, body: metadata, json: true}, function(err) {
-      console.error(err);
+      if(err) { console.error(err); }
     });
     break;
   case 'put':
     request.put({url: webhook.url, body: metadata, json: true}, function(err) {
-      console.error(err);
+      if(err) { console.error(err); }
     });
     break;
   case 'patch':
     request.del({url: webhook.url, body: metadata, json: true}, function(err) {
-      console.error(err);
+      if(err) { console.error(err); }
     });
     break;
   case 'delete':
     request.del({url: webhook.url, json: true}, function(err) {
-      console.error(err);
+      if(err) { console.error(err); }
     });
     break;
   // get
   default:
     request(webhook.url, function(err) {
-      console.error(err);
+      if(err) { console.error(err); }
     });
   }
 

